test/lint/hashes.rb

```diff
@@ -227,5 +227,18 @@ def test_hscan
       expected = ['0', [%w[f1 Jack], %w[f2 33]]]
       assert_equal expected, redis.hscan('foo', 0)
     end
+
+    def test_hexpire
+      target_version "7.4.0" do
+        r.hset("foo", "f1", "v2")
+
+        assert_equal [1], r.hexpire("foo", 1, "f1")
+        assert r.hexists("foo", "f1")
+
+        sleep(1)
```

Comment:
Sleeping in test suite is always a bad idea.

A better way to test `hexpire` is to implement `httl`, and use it to check the TTL changed.